spec/cedar_policy/authorizer_spec.rb

```
@@ -172,5 +172,152 @@
+              resource: [Image]
+          };
+        SCHEMA
+      end
+
+      let(:entities) do
+        CedarPolicy::Entities.new(entities_data, schema: schema)
+      end
+
+      describe "User can View image" do
+        let(:action) { CedarPolicy::EntityUid.new("Action", "View") }
+        let(:entities_data) {
```

rubocop (1.68.0)

[convention] Avoid using `{...}` for multi-line blocks.

(Style/BlockDelimiters)